### ISSUE TITLE
Port to 'Bootstrap Vue'

### ISSUE THREAD

**Issue by gfournier** [mariobuikhuizen/ipyvuetify#12]
Here is the port for Bootstrap Vue, it's located in this fork (`bootstrapvue` branch) and available in PyPi: https://github.com/gfournier/ipyvuetify/tree/bootstrapvue 

Major parts of the code are standard to Vue (and not specific to Vuetify), maybe we can move on from this point.

poke @maartenbreddels @mariobuikhuizen 

Code generator for 'Bootstrap Vue' is located here: https://github.com/gfournier/bootstrap-vue-api-generator

**Comment by saraedum** [comment]
@gfournier Have you made any attempts to pull out the bits that are not specific to Vuetify? I would like to try as I need the general infrastructure for another project.

**Comment by gfournier** [comment]
We plan with @mariobuikhuizen to extract the part that is generic in another project (ipyvue). The goal of such a project would be to expose ipywidgets libraries from Vue component libraries - for use in Jupyter notebooks in link with Voila templates.

**Comment by saraedum** [comment]
Great. Ping me if you want any help with that :)

**Comment by maartenbreddels** [comment]
Cool, help is welcome, we're also planning on making a cookiecutter (and maybe a tool based on the autogenerate code) for people to make new vue based widgets very quickly. 
I think the best way forward is we start it a few weeks from now, we'll ping you when that starts.

**Comment by saraedum** [comment]
Sounds good. In the meantime, I might just try to work on the things here a bit, e.g., to document how to make this work in Jupyter Lab, get Hot Module Reloading, and things like that if you don't mind.

**Comment by maartenbreddels** [comment]
Yes, feel free to, and feel free to reach out to us as well, we are mostly active on gitter https://gitter.im/jupyter-widgets/Lobby

**Comment by mariobuikhuizen** [comment]
ipyvue is now available on https://github.com/mariobuikhuizen/ipyvue.

The changes to ipyvuetify to support ipyvue can be found here: #30.

Ping @gfournier @saraedum
